com.dynamo.cr/com.dynamo.cr.bob/src/com/dynamo/bob/Project.java

```diff
@@ -509,79 +508,14 @@ public Task<?> createTask(IResource inputResource, Class<? extends Builder<?>> b
-
-                BuilderParams p1 = b1.getAnnotation(BuilderParams.class);
-                BuilderParams p2 = b2.getAnnotation(BuilderParams.class);
-
-                return p1.createOrder() - p2.createOrder();
-            }
-        });
-        return sortedInputs;
-    }
-
-    /*
-        The same logic implemented in the Editor.
-        If you change something here, make sure you change it in resource.clj
-        (defignore-pred).
-    */
-    private void loadIgnoredFilesAndFolders() throws CompileExceptionError {
-        String excludeFoldersStr = this.option("exclude-build-folder", "");
-        List<String> excludeFolders = BundleHelper.createArrayFromString(excludeFoldersStr);
-        excluedFilesAndFoldersEntries.addAll(excludeFolders);
-        List<String> defIgnoreEntries = new ArrayList<String>();
-        final File defIgnoreFile = new File(getRootDirectory(), ".defignore");
-        if (defIgnoreFile.isFile()) {
-            try {
-                defIgnoreEntries = FileUtils.readLines(defIgnoreFile, "UTF-8")
-                        .stream()
-                        .filter(s -> !s.isEmpty())
-                        .collect(Collectors.toList());
-            }
-            catch(IOException e) {
-                throw new CompileExceptionError("Unable to read .defignore", e);
-            }
-        }
-        excluedFilesAndFoldersEntries.addAll(defIgnoreEntries);
-        // remove initial "/" from excluded folder names
-        for(int i = 0; i < excluedFilesAndFoldersEntries.size(); i++) {
-            String entry = excluedFilesAndFoldersEntries.get(i);
-            if (entry.startsWith("/")) {
-                excluedFilesAndFoldersEntries.set(i, entry.substring(1));
-            }
-        }
-    }
-
     private void createTasks() throws CompileExceptionError {
         tasks = new HashMap<String, Task<?>>();
-        List<String> sortedInputs = sortInputs(); // from findSources
-
-        // To currently know the output resources, we need to parse the main.collectionc
-        // We would need to alter that to get a correct behavior (e.g. using GameProjectBuilder.findResources(this, rootNode))
-
-        // create tasks for inputs that are not excluded
-        for (String input : sortedInputs) {
-            boolean skipped = false;
-            // Ignore for resources.
-            // Check comment for loadIgnoredFilesAndFolders()
-            for (String excludeEntry : excluedFilesAndFoldersEntries) {
-                if (input.startsWith(excludeEntry)) {
-                    skipped = true;
-                    break;
-                }
-            }
-            if (!skipped) {
-                Class<? extends Builder<?>> builderClass = getBuilderFromExtension(input);
-                if (!ignoreTaskAutoCreation.contains(builderClass)) {
-                    Task<?> task = createTask(input, builderClass);
-                }
+        if(this.inputs == null || this.inputs.isEmpty()) {
```

Comment:
If there is no inputs specified (used for tests to build separate files) we start creation from `game.project` file, otherwise build specified inputs one by one